From d9dc3b0ef84276cb10f4f665ee7139ad935c3998 Mon Sep 17 00:00:00 2001
From: icg_intg <icg_intg@intel.com>
Date: Fri, 9 Nov 2018 09:58:28 +0800
Subject: [PATCH 2269/2278] isys & psys package lib2600b0 for commit-id 88ddf58

FW_Manifest_ID: 8929d5195a8b49989d228005908cfd2bdabac0de

FW_Build_Branch: stable_iot_mr4_pv_r1_20180817

FW_Build_Topic: auto_merge_queue

library version: commit-id 88ddf58

FW Bundle patch number of this release :  367544

FW tag : scci_stable_iot_mr4_pv_r1_20180817_20181107_1236

Change-Id: I504f6aaf1e7d8c84bbc4132b787975c2e965c8ac
Depends-on: I91dc483949ef88e2bdc094e33fd60ad1e744e28e
Signed-off-by: icg_intg <icg_intg@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h           | 2 +-
 .../lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h        | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
index fd7dc00..50ec8e9 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20181024
+#define IA_CSS_FW_PKG_RELEASE  0x20181107
diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
index fd7dc00..50ec8e9 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20181024
+#define IA_CSS_FW_PKG_RELEASE  0x20181107
-- 
2.7.4

